### DISCUSSION TITLE
I am getting this error. Property 'css' is missing in type while updating the paste version

### DISCUSSION THREAD

**Discussion by Ghanshyam-K-Dobariya** [twilio-labs/paste#2424] [Q&A] [answered]
Hi Team,

I updated my paste version from 10.6.0 to 10.20.0
& twilio-paste/icons from 5.3.0  to 5.9.1

Now when I build on local, I get this error

```
TS2741: Property 'css' is missing in type 

'{ children: Element[]; onChange: (val: any) => void; legend: string; name: string; value: string; required: true; }' 

but required in type 'Pick<RadioGroupProps, "value" | "children" | "form" | "legend" | "slot" | "style" | "title" | "element" | "color" | "defaultChecked" | "defaultValue" | "suppressContentEditableWarning" | ... 252 more ... | "errorText">'.
```

**Comment by nkrantz** [comment]
Hey @Ghanshyam-K-Dobariya - looks like you're not on the most recent Paste versions. Not sure exactly what caused your errors in your `10.6.0` --> `10.20.0` update, but I'd recommend getting on the most recent versions (`twilio-paste/core: 13.0.0` & `twilio-paste/icons: 8.0.0`). Let us know if you're still seeing the error after updating!

**Comment by andioneto** [comment]
Hi there @Ghanshyam-K-Dobariya 👋 

Could you please share your `tsconfig.json` for the project in question? We have a [draft of a pull request](https://github.com/twilio-labs/paste/pull/2443) up now to address this, once we complete some more testing, we will follow up with a status update.

I can probably help provide a temporary fix as well, but without knowing specifics about your TS config (e.g. version and config options), I can't deterministically say.

Here is a quick summary of related information:

- this is directly related to our upgrade of our EmotionJS dependencies that was released in `@twilio-paste/core@13`
- this can be traced to internal changes to `@emotion/react@^11` 
- folks will temporarily be conditionally impacted in development if your project configuration meets the characteristics [described in the v11 release summary](https://emotion.sh/docs/emotion-11#css-prop-types)

More coming soon, please stay tuned. 📻
Thank you for your patience and thank you for taking time to improve Paste! 🥇

**Reply by loreina** [reply]
Hi @Ghanshyam-K-Dobariya, just follow up in the thread here. Would you be able to share your `tsconfig.json` with us? Thanks!

**Comment by SiTaggart** [comment, marked as answer]
Hi @Ghanshyam-K-Dobariya 

A new version of Paste went out on Friday, Core 13.0.1. We are confident that this issue should resolve itself by upgrading to that patch version.

Let us know how you go with it.
